CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# origin 0.19
+
+## Changes
+
+The `origin-program`, `origin-thread`, and `origin-signal` features are now
+combined into a single `take-charge` feature.
```

Comment:
These changes were done after the 0.19 release. Not as part of it.

Reply:
Oops, fixed.